packages/f-checkout/test/specs/component/f-checkout.component.spec.js

```diff
@@ -33,30 +32,28 @@ describe('f-checkout component tests', () => {
     });
 
     it('should display times in ascending order, with default text "As soon as possible" showing first', () => {
-        // Act 
+        // Act
         CheckoutComponent.selectOrderTime('As soon as possible');
-        
+
         // Assert
         expect(CheckoutComponent.isOrderTimeDropdownDisplayed()).toBe(true);
         expect(CheckoutComponent.getOrderTimeOptionText(0)).toBe('As soon as possible');
         expect(CheckoutComponent.getOrderTimeOptionText(1)).toBe('Monday 00:15');
         expect(CheckoutComponent.getOrderTimeOptionText(2)).toBe('Monday 00:30');
     });
 
-    it.only('should prevent user from writing a note of over 200 characters', () => {
-        // Arrange 
-        const userNote = 'A'
+    it('should prevent user from writing a note of over 200 characters', () => {
+        // Arrange
+        const userNote = 'A';
         const addressInfo = {
             note: userNote.repeat(300)
         };
 
         // Act
-        CheckoutComponent.inputUserNote(addressInfo)
```

Comment:
do we not need these functions? I quite like the input from the user as well as getting the maxlength attribute, but happy to discuss! :)

Reply:
Had a chat with Kevin and Ben about this and the expect is already testing for it to be 200 and also we're looking at revisiting this as a unit test.

Reply:
ahh okay cool, sounds good! Might need to remove the functions from the component object too if we're getting rid, or this could be tackled separately if it's becoming a unit test 👍